FEXHeaderUtils/FEXHeaderUtils/Syscalls.h

```diff
@@ -28,6 +28,10 @@ namespace FHU::Syscalls {
 #define CLONE_PIDFD 0x00001000
 #endif
 
+#ifndef SYS_statx
+#define SYS_statx 291
```

Comment:
This is only viable on AArch64. This would break x86-64 host environments where statx syscall is 332.